README.md

```diff
@@ -1,2 +1,68 @@
-# MediaArea-Website-New
-MediaArea Website
+# MediaArea-Website
+
+[![Build Status](https://travis-ci.org/MediaArea/MediaArea-Website.svg?branch=master)](https://travis-ci.org/MediaArea/MediaArea-Website)
+
+# How to install
+
+## Dependencies
+
+* Apache web server >= 2.2 (should also work on nginx and lighttpd)
+    * mod rewrite is recommended
+* Php >= 7.0
+    * Mandatory packages for debian-like distributions : libapache2-mod-php7.0, php7.0-cli, php7.0-intl, php7.0-curl, php7.0-opcache
```

Comment:
directly the command ("apt ..." and "yum ...") as it is done with "git clone ...".